pkgs/tools/networking/boundary/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, lib, fetchurl, unzip }:
+let
+  version = "0.1.4";
+
+  system = stdenv.hostPlatform.system;
+  suffix = {
+    x86_64-linux = "linux_amd64";
+    aarch64-linux = "linux_arm64";
+    x86_64-darwin = "darwin_amd64";
+  }."${system}" or (throw "Unsupported system: ${system}");
+
+  fetchsrc = sha256: fetchurl {
+    url = "https://releases.hashicorp.com/boundary/${version}/boundary_${version}_${suffix}.zip";
+    sha256 = sha256."${system}";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "boundary";
+  inherit version;
+
+  src = fetchsrc {
+    x86_64-linux = "14f4mc5l04ammszvcq7dv8a6pvdz13k5y6r226lqs72lfyxi5kc7";
+    aarch64-linux = "1blzn42birpq2pbqz63ffzd0p2xcgrgmg2q6jmmabfjzmp0z7ax2";
+    x86_64-darwin = "03w1c3p45nz1h3cf705d3n15c4rxsf1r448lcqkxaarsicxs32gw";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+  nativeBuildInputs = [ unzip ];
+  unpackPhase = "unzip $src";
+  installPhase = "install -D boundary $out/bin/boundary";
```

Comment:
```suggestion
  installPhase = ''
    install -D boundary $out/bin/boundary
  '';
```

Reply:
Why use multiple lines when it's a single command?

Reply:
It makes it easier to add commands, spot that it is a command and not some setting or env and adds a final new line incase wrapper or hooks did not handle that quite right. Also treewide changes are always easier if there are not multiple variants of essentially the same.

Reply:
Ok fixed